### PR TITLE
Add cloud cost env variables

### PR DESCRIPTION
**Pull request by Sean-Holcomb**
## Does this PR rely on any other PRs?

- 
- 

## Links to Issues or ZD tickets this PR addresses or fixes

- 
- 

## How was this PR tested?
Tested by with `helm template`

### CODE REVIEW

File: cost-analyzer/templates/cost-analyzer-deployment-template.yaml [hunk 1]
```diff
@@ -712,6 +712,18 @@ spec:
               value: {{ (quote .Values.kubecostModel.etlAssetReconciliationEnabled) | default (quote true) }}
             - name: ETL_USE_UNBLENDED_COST
               value: {{ (quote .Values.kubecostModel.etlUseUnblendedClost) | default (quote false) }}
+            {{- with .Values.kubecostModel.cloudCost }}
+            - name: CLOUD_COST_ENABLED
+              value: {{ (quote .enabled) | default (quote false) }}
+            {{- with .labelList }}
```
Comment (nikovacevic) [inline cost-analyzer/templates/cost-analyzer-deployment-template.yaml:718]: This doesn't have to be `.Values.kubecostModel.cloudCost.labelList`?

Comment (nikovacevic) [reply cost-analyzer/templates/cost-analyzer-deployment-template.yaml:718]: etc. for the others

Comment (nikovacevic) [reply cost-analyzer/templates/cost-analyzer-deployment-template.yaml:718]: Might just be helm syntax with which I'm not familiar.

Comment (Sean-Holcomb) [reply cost-analyzer/templates/cost-analyzer-deployment-template.yaml:718]: it is, within the {{with}} block the specified path becomes the relative path, if the path doesn't exist, then the whole block doesn't run

Comment (nikovacevic) [reply cost-analyzer/templates/cost-analyzer-deployment-template.yaml:718]: Nifty

**Review by nikovacevic — APPROVED** [review]

**Pull request merged**